pkg/signer/memory.go

```diff
@@ -30,12 +30,25 @@ import (
 	tsx509 "github.com/sigstore/timestamp-authority/pkg/x509"
 )
 
+func getCurveFromSigner(signer crypto.Signer) (elliptic.Curve, error) {
```

Comment:
memory signer hardcodes the signer's curve - https://github.com/sigstore/timestamp-authority/blob/403a0a1fc1944a77ff4c4c3fa528f3d90c518d35/pkg/signer/signer.go#L42 - so this isn't needed

I think this is reasonable, as it's only meant for testing.